### PR TITLE
fix(init): init package with better main, types

### PR DESCRIPTION
**Pull request by ofrobots**
Fixes: https://github.com/google/ts-style/issues/144

### CODE REVIEW

File: src/init.ts [hunk 1]
```diff
@@ -210,17 +230,16 @@ export async function init(options: Options): Promise<boolean> {
     }
 
     try {
-      // TODO(ofrobots): add proper error handling.
-      cp.spawnSync('npm', ['init', '-y']);
-      packageJson = await readJson('./package.json');
+      packageJson = DEFUALT_PACKAGE_JSON;
```
Comment (kjin) [inline src/init.ts]: Do we still need the try-catch here?

File: src/init.ts [hunk 2]
```diff
@@ -21,7 +21,19 @@ import * as path from 'path';
 import {Options} from './cli';
 import {readFilep as read, readJsonp as readJson, writeFileAtomicp as write} from './util';
 
-const pkg = require('../../package.json') as PackageJson;
+const pkg = require('../../package.json');
```
Comment (kjin) [inline src/init.ts]: Why did the type annotation here disappear?

Comment (ofrobots) [reply src/init.ts]: That wasn't an annotation but rather a cast. The cast was not necessary.

Comment (ofrobots) [reply src/init.ts]: Also – we don't use this yet – but going forward we would stronger types for package.json available from new resolveJsonModule in TS 2.9.

File: src/init.ts (hunk 1 above)
Comment (ofrobots) [reply src/init.ts]: Indeed not. Removed.

**Pull request merged**